docs/data-cookbooks/cnm-workflow.md

```diff
@@ -233,11 +235,13 @@ Below is a description of each of these tasks:
 
 You can also manipulate the data sent to downstream tasks using `task_config` for various states in your workflow resource configuration. Read more about how to configure data on the [Workflow Input & Output](https://nasa.github.io/cumulus/docs/workflows/input_output) page.
 
+Documentation for `CNMToCMA` Lambda package is provided in the [source repository](https://git.earthdata.nasa.gov/projects/POCUMULUS/repos/cnmtogranule/browse).
+
 ##### CnmResponse
 
-The `CnmResponse` Lambda generates a CNM response message and puts it on a the `CNMResponseStream` Kinesis stream.
+The `CnmResponse` Lambda generates a CNM response message and puts it on the `response-endpoint` Kinesis stream.
 
-The `CnmResponse` Lambda package is provided (as of release 1.8) in the `cumulus-data-shared` bucket, with documentation provided in the [source repository](https://git.earthdata.nasa.gov/projects/POCUMULUS/repos/cnmresponsetask/browse).
+Documentation for `CnmResponse` Lambda package is provided in the [source repository](https://git.earthdata.nasa.gov/projects/POCUMULUS/repos/cnmresponsetask/browse).
```

Comment:
This note about the `cumulus-data-shared` bucket seemed a bit out of place and redundant given that we point people to the task definition TF files above